lib/tara/installer.rb

```diff
@@ -63,6 +76,50 @@ def bundle_gems
       end
     end
 
+    def find_installed_gems
+      Bundler.with_clean_env do
+        definition = Bundler::Definition.build('lib/vendor/Gemfile', 'lib/vendor/Gemfile.lock', false)
+        @without_groups.each do |group|
+          definition.add_optional_group(group)
+        end
```

Comment:
The reason it works in Puck is because the copying is executed in an isolated scripting container. Bundler has a global state that is locked on first access, so changing `BUNDLE_WITHOUT` has no effect during run time.

I will investigate alternatives...